src/components/VideoGrid/VideoGrid.component.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import './VideoGrid.styles.scss';
+import VideoCard from '@components/VideoCard';
+import Button from '@components/Button';
+
+function VideoGrid(props) {
+  return (
+    <div className="video-grid__container">
+      <div className={'video-grid ' + props.className}>
+        {props.items.map((item) => (
+          <VideoCard key={item.id.videoId} item={item} onClick={() => {}} />
+        ))}
+      </div>
+      {props.loadMore ? (
+        <div className="video-grid__load-more-row">
+          <Button text="Load more" onClick={props.loadMore} />
+        </div>
+      ) : null}
```

Comment:
Ass React DOM doesn't display falsy values, you could use the `&&` operator here.

Reply:
I didn't know that, thanks.